src/scene/shader-lib/chunks/lit/frag/shadowStandard.js

```diff
@@ -62,6 +62,26 @@ float getShadowSpotPCF3x3(SHADOWMAP_ACCEPT(shadowMap), vec3 shadowCoord, vec4 sh
     return _getShadowPCF3x3(SHADOWMAP_PASS(shadowMap), shadowCoord, shadowParams.xyz);
 }
 
+float _getShadowPCF1x1(SHADOWMAP_ACCEPT(shadowMap), vec3 shadowCoord, vec3 shadowParams) {
+    float z = shadowCoord.z;
+    vec2 uv = shadowCoord.xy * shadowParams.x; // 1 unit - 1 texel
+    float shadowMapSizeInv = 1.0 / shadowParams.x;
+    vec2 base_uv = floor(uv + 0.5);
```

Comment:
it seems you're forcing pixel snapping / no filtering here, which I don't think you should? This function should look similarly to what PCF1 shadow does for clustered lights, which is a lot cheaper:

```
    float getShadowSpotClusteredPCF1(SHADOWMAP_ACCEPT(shadowMap), vec3 shadowCoord, vec4 shadowParams) {
        return textureShadow(shadowMap, shadowCoord);
    }

```

Reply:
I've updated this to be in line with clustered light shadows.